1_space_odyssey/js/sander-9000/app.js

```diff
@@ -100,23 +102,29 @@ var canRun = true;
 var canFly = false;
 var hasWeapon = true;
 var isInGoodMood = true;
+*/
+
+var android = {
+  isAlive: true,
```

Comment:
nice one.

you can take it a step further:
` - Your android needs a head, upperBody, lowerBody & [ anyThingElseYouCanDreamOf ].`

```js
var obj = {
     property: {
        ...
     },
     property: {
        ...
     },
     property: {
        ...
     },
}
```